content/blog/contributor-highlight-francesco-canovai/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Contributor Spotlight: Francesco Canovai"
+date: 2025-05-07T14:51:43+02:00
+draft: false
+image:
+    url: 
+    attribution:
+author: fdrees
+tags:
+ - testing
+ - github
+ - kubernetes
+ - postgresql
+ - open-source
+summary: "In a mini-series on this blog we highlight the work of the community.
+  Today we meet Francesco Canovai, Senior Software Development Engineer at EDB, 
+  and CloudNativePG maintainer." 
+---
+
+# Contributor Spotlight: 
+
+Building and maintaining and open source project takes a village. In a
+mini-series on this blog we would like to highlight the work of our
+maintainers, component owners, and members of the larger community.
+
+Today it's my great pleasure to introduce [Francesco Canovai](https://github.com/fcanovai/), 
+my colleage at EDB. Francesco is a Senior Software Development Engineer, 
+living in Prato, Italy. No point asking when he first found out about 
+CloudNativePG, he was in the room when/where the project started! The 
+first commit with his name on it is the initial suite of e2e tests.
```

Comment:
From experience presenting materials about the project: we get questions on what "e2e test" means.
I'd say something like "integration tests", or if not at least expand e2e as end-to-end tests.